build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ReplaceFileContents.cs

```diff
@@ -32,7 +32,8 @@ public override bool Execute ()
 				}
 			}
 
-			File.Delete (DestinationFile.ItemSpec);
+			if (File.Exists (DestinationFile.ItemSpec))
+				File.Delete (DestinationFile.ItemSpec);
```

Comment:
I think the only reason you would need this is if the directory did not exist. Was this throwing `DirectoryNotFoundException`?

`File.Delete()` shouldn't throw if the file doesn't exist:

https://docs.microsoft.com/dotnet/api/system.io.file.delete#remarks

Reply:
Yeah I was using this with a path where the destination directory didn't exist, I figured this was easier than checking if the directory existed.